web/src/app/shared/components/projects-list/projects-list.component.ts

```diff
@@ -1,15 +1,25 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnChanges } from '@angular/core';
 
 // Dashboard hub model
-import { ProjectModel } from '../../models/index.model';
+import { PingService } from '../../../core/services/index.service';
+import { PingModel, ProjectModel } from '../../models/index.model';
 
 @Component({
   selector: 'dashboard-projects-list',
   templateUrl: './projects-list.component.html',
   styleUrls: ['./projects-list.component.scss'],
 })
-export class ProjectsListComponent {
+export class ProjectsListComponent implements OnChanges {
 
   @Input()
   public projects: ProjectModel[] = [];
+  public pingCount: [];
+
+  constructor(private pingService: PingService) { }
+
+  ngOnChanges(): void {
+    this.projects.forEach((project: ProjectModel) => {
+      this.pingService.findAllPingsCount(project.uid).subscribe((data: PingModel[]) => project.pingCount = data.length);
```

Comment:
Following my above comment, I then think in the html can have a call to this method on the model...

```typescript
public getTotalPings(): number {
   let total: number = 0;
   this.monitors.forEach((monitor: MonitorModel) => total += monitor.successfulPings + monitor.unsuccessfulPings);

    return total;
}
```